scripts/configure_invokeai.py

```diff
@@ -578,23 +578,32 @@ def select_outputs(root:str,yes_to_all:bool=False):
     completer.set_default_dir(os.path.expanduser('~'))
     completer.complete_extensions(())
     completer.set_line(default)
-    return input('Select the default directory for image outputs [{default}]: ')
+    return input(f'Select the default directory for image outputs [{default}]: ') or default
 
 #-------------------------------------
 def initialize_rootdir(root:str,yes_to_all:bool=False):
     assert os.path.exists('./configs'),'Run this script from within the top level of the InvokeAI source code directory, "InvokeAI"'
     
     print(f'** INITIALIZING INVOKEAI RUNTIME DIRECTORY **')
-    root = root or select_root(yes_to_all)
-    outputs = select_outputs(root,yes_to_all)
-    Globals.root = root
-        
-    print(f'InvokeAI models and configuration files will be placed into {root} and image outputs will be placed into {outputs}.')
-    print(f'\nYou may change these values at any time by editing the --root and --output_dir options in "{Globals.initfile}",')
+    root_selected = False
+    while not root_selected:
+        root = select_root(root,yes_to_all)
+        outputs = select_outputs(root,yes_to_all)
+        Globals.root = os.path.abspath(root)
+        outputs = outputs if os.path.isabs(outputs) else os.path.abspath(os.path.join(Globals.root,outputs))
+
+        print(f'\nInvokeAI models and configuration files will be placed into "{root}" and image outputs will be placed into "{outputs}".')
+        if not yes_to_all:
+            root_selected = yes_or_no('Accept these locations?')
+        else:
+            root_selected = True
+
+    print(f'\nYou may change the chosen directories at any time by editing the --root and --output_dir options in "{Globals.initfile}",')
```

Comment:
Typo: `--output_dir` to `--outdir`

```suggestion
    print(f'\nYou may change the chosen directories at any time by editing the --root and --outdir options in "{Globals.initfile}",')
```